pkg/proxy/nftables/proxier_test.go

```diff
@@ -494,6 +494,9 @@ func TestOverallNFTablesRules(t *testing.T) {
 		}),
 	)
 
+	// populate serviceCIDRs
+	fp.OnServiceCIDRsChanged([]string{"172.30.0.0/24", "fd00:10:96::/112"})
```

Comment:
ok, super nit-picky, but a `/16` IPv4 service cidr would be a lot more normal

also, maybe just set `proxier.serviceCIDRs` explicitly in `NewFakeProxier` rather than doing this in each unit test?